.rubocop.yml

```diff
@@ -80,7 +80,6 @@ Metrics/ClassLength:
 
 Layout/LineLength:
   Max: 150
-  IgnoredPatterns: ['\A#']
```

Comment:
Shouldn't this pattern be an `AllowedPattern` then?

Reply:
Yes, it could have been renamed to `AllowedPattern`.

But the pattern line can be removed: RuboCop doesn't complain any more about long comment lines.